docs/calicoctl/endpoint.md

```diff
@@ -162,6 +162,11 @@ calicoctl endpoint <ENDPOINT_ID> profile (append|remove|set) [--host=<HOSTNAME>]
     <PROFILES>: One or more profiles to append, remove, or set on the endpoint.
 ```
 
+This command perform the desired action even if the `<ENDPOINT_ID>` is specified 
```

Comment:
But i'm not too fussed either way.

Wording is a little wrong as it stands though...   "This command perform"   missing s?